test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -11163,6 +11158,20 @@ def revert_to_defaults(self):
         self.resource_group = None
         return self.isUp()
 
+    def revert_server_attribs(self):
```

Comment:
I dislike this name since it could be confused with revert_to_defaults().  How about something more specific: revert_server_tearDown() or something like that.